type assertion failure right after startup

Trying to run this against salt 2017.7.7 but I'm getting a type assertion failure right after startup. Not sure if this is due to old salt or what, hoping you can provide some insight? I can provide a fix assuming you're open to it but if this is a version thing then I'm sure there's probably more things like this to find too.

```
panic: interface conversion: interface {} is string, not []uint8

goroutine 9 [running]:
github.com/kpetremann/salt-exporter/pkg/parser.Event.Parse({_}, _)
	/home/runner/work/salt-exporter/salt-exporter/pkg/parser/parser.go:98 +0xa6c
github.com/kpetremann/salt-exporter/pkg/listener.(*EventListener).ListenEvents(0xc0000760f0)
	/home/runner/work/salt-exporter/salt-exporter/pkg/listener/listener.go:127 +0x16f
created by main.start in goroutine 1
	/home/runner/work/salt-exporter/salt-exporter/cmd/salt-exporter/main.go:72 +0x467
```

--- a/pkg/parser/parser.go
+++ b/pkg/parser/parser.go
@@ -95,7 +95,13 @@ func statemoduleResult(event event.SaltEvent) *bool {
 
 // ParseEvent parses a salt event.
 func (e Event) Parse(message map[string]interface{}) (event.SaltEvent, error) {
-	body := string(message["body"].([]byte))
+	var body string
+
+	if raw, ok := message["body"].([]byte); ok {
+		body = string(raw)
+	} else {
+		body = message["body"].(string)
+	}
 	lines := strings.SplitN(body, "\n\n", 2)
 
 	tag := lines[0]

